libs/components/modals/src/lib/modules/confirm/confirm.component.spec.ts

```diff
@@ -341,4 +341,228 @@ describe('Confirm component', () => {
+        '.sky-confirm-buttons .sky-btn'
+      );
+
+      await expectAsync(fixture.nativeElement).toBeAccessible();
+      buttons[0].click();
+    });
+
+    it('should be accessible when displaying a custom confirm', async () => {
+      const fixture = createConfirm({
+        message: 'confirm message',
+        type: SkyConfirmType.Custom,
+        buttons: [
+          {
+            text: 'Custom label',
+            action: 'foo',
+          },
+        ],
+      });
+
+      fixture.detectChanges();
+      const buttons = fixture.nativeElement.querySelectorAll(
+        '.sky-confirm-buttons .sky-btn'
+      );
+
+      await expectAsync(fixture.nativeElement).toBeAccessible();
+      buttons[0].click();
+    });
+
+    it('should be accessible when displaying a custom confirm with body', async () => {
+      const fixture = createConfirm({
+        message: 'confirm message',
+        body: 'additional text',
+        type: SkyConfirmType.Custom,
+        buttons: [
+          {
+            text: 'Custom label',
+            action: 'foo',
+          },
+        ],
+      });
+
+      fixture.detectChanges();
+      const buttons = fixture.nativeElement.querySelectorAll(
+        '.sky-confirm-buttons .sky-btn'
+      );
+
+      await expectAsync(fixture.nativeElement).toBeAccessible();
+      buttons[0].click();
+    });
+
+    it('should be accessible when displaying a custom confirm with all button types', async () => {
+      const fixture = createConfirm({
+        message: 'confirm message',
+        body: 'additional text',
+        type: SkyConfirmType.Custom,
+        buttons: [
+          {
+            text: 'Custom label',
+            action: 'foo',
+          },
+          {
+            text: 'Custom label',
+            action: 'bar',
+            styleType: 'primary',
+          },
+          {
+            text: 'Custom label',
+            action: 'buz',
+            styleType: 'default',
+          },
+          {
+            text: 'Custom label',
+            action: 'baz',
+            styleType: 'link',
+          },
+        ],
+      });
+
+      fixture.detectChanges();
+      const buttons = fixture.nativeElement.querySelectorAll(
+        '.sky-confirm-buttons .sky-btn'
+      );
+
+      await expectAsync(fixture.nativeElement).toBeAccessible();
+      buttons[0].click();
+    });
+
+    it('should be accessible when autofocus is specified on a button from the config', async () => {
+      const fixture = createConfirm({
+        message: 'confirm message',
+        type: SkyConfirmType.Custom,
+        buttons: [
+          {
+            text: 'foo',
+            action: 'foo',
+          },
+          {
+            text: 'bar',
+            action: 'bar',
+          },
+          {
+            text: 'baz',
+            action: 'baz',
+            autofocus: true,
+          },
+        ],
+      });
+
+      fixture.detectChanges();
+
+      const buttons = fixture.nativeElement.querySelectorAll(
```

Comment:
There's a lot of repeated code in these tests. Maybe you could create a utility function that could be shared by all?
```
async function verifyAccessibility(options: {...}): Promise<void> {
  const fixture = createConfirm(options);

  fixture.detectChanges();

  const buttons = fixture.nativeElement.querySelectorAll(
    '.sky-confirm-buttons .sky-btn'
  );

  await expectAsync(fixture.nativeElement).toBeAccessible();
  
  buttons[0].click();
}

await verifyAccessibility({
  message: 'confirm message',
  body: 'additional text',
  type: SkyConfirmType.Custom,
  buttons: [
    {
      text: 'Custom label',
      action: 'foo',
    },
  ],
});
```

Reply:
Good call. Done